datumaro/components/media.py

```diff
@@ -112,40 +156,48 @@ def data(self) -> Optional[AnyData]:
             return self._data()
         return self._data
 
+    @property
+    def bytes(self) -> Optional[bytes]:
+        if self.has_data:
+            if callable(self._data):
+                _bytes = self._data()
+            _bytes = self._data
+            if isinstance(_bytes, bytes):
+                return _bytes
+        return None
+
+    @property
+    def has_data(self) -> bool:
+        return self._data is not None
+
     def __repr__(self) -> str:
-        return f"{self.__class__.__name__}(data={self._data})"
+        return f"{self.__class__.__name__}(data=" + repr(self._data)[:20].replace("\n", "") + "...)"
```

Comment:
Is `replace("\n", "")` general when representing data?

Reply:
Removing `\n` makes things a bit prettier in one line. There would be no problem for any sorts of data.